software_scale_context_test.go

```diff
@@ -0,0 +1,104 @@
+package astiav_test
+
+import (
+	"image"
+	"testing"
+
+	"github.com/asticode/go-astiav"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSoftwareScaleContext(t *testing.T) {
+	f1 := astiav.AllocFrame()
+	require.NotNil(t, f1)
+	defer f1.Free()
+
+	f2 := astiav.AllocFrame()
+	require.NotNil(t, f2)
+	defer f2.Free()
+
+	f3 := astiav.AllocFrame()
+	require.NotNil(t, f3)
+	defer f3.Free()
+
+	srcW := 100
+	srcH := 100
```

Comment:
Could you make the `srcH` different than `srcW` (`150` for instance) and `dstH` different then `dstW` (`250` for instance)?